webapp/src/components/modals/channel_settings/edit_channel_settings.test.tsx

```diff
@@ -367,6 +366,9 @@ describe('components/EditChannelSettings', () => {
         expect(wrapper.exists('#jira-delete-subscription')).toBe(true);
         wrapper.find('#jira-delete-subscription').simulate('click');
 
+        expect(wrapper.state().showConfirmModal).toBe(true);
+        wrapper.instance().handleConfirmDelete();
```

Comment:
This seems a little redundant since the previous test already tests for error (I introduced this though). 0/5 I suggest removing the check for failure in the previous test, or you can remove this test and keep the error check in the other.

Reply:
I'm going to remove the check for failure in the previous test. The name of this test suggests the check is in here.